ground/src/main/java/com/google/android/ground/ui/map/CameraPosition.kt

```diff
@@ -62,3 +63,7 @@ data class CameraPosition(
     }
   }
 }
+
+// TODO(#1712): Remove [isAllowZoomOut] and [bounds] from CameraPosition class.
+  fun com.google.android.gms.maps.model.CameraPosition.toModelObject(isAllowZoomOut: Boolean, bounds: Bounds) =
```

Comment:
I know it's not a new addition, but should isAllowZoomOut be named allowZoomOut or isZoomOutAllowed? The current name feels a little clunky 😅

Reply:
Hey @JSunde, this PR isn't ready for review yet, it's just a WIP draft. isAllowZoomOut should be removed completely.